apps/extension/src/webview/index.tsx

```
@@ -0,0 +1,1709 @@
+		case 'SET_ERROR':
+			return { ...state, error: action.payload, loading: false };
+		case 'CLEAR_ERROR':
+			return { ...state, error: undefined };
+		case 'INCREMENT_REQUEST_ID':
+			return { ...state, requestId: state.requestId + 1 };
+		case 'UPDATE_TASK_STATUS':
+			const updatedTasks = state.tasks.map((task) =>
+				task.id === action.payload.taskId
+					? { ...task, status: action.payload.newStatus }
+					: task
+			);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)